airgun/views/os.py

```diff
@@ -35,6 +37,19 @@ def read(self):
             self.browser.click(self.view_tab)
             return self.ptables.read()
 
+    @View.nested
+    class parameters(View):
```

Comment:
Please take a look at widgetastic_patternfly `Tab()` view, we should probably use it or make our equivalent instead. Or submit an issue if you think it's out of scope :)

Reply:
https://github.com/SatelliteQE/airgun/issues/35